_posts/2021-08-23-gsoc-2021-project.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: GSoC 2021 Project
```

Comment:
```suggestion
title: Google Summer of Code 2021 - Dask Project
```

GSOC is not likely to be an acronym most people know, better to spell it out in full.